pkg/quota/evaluator/core/persistent_volume_claims.go

```diff
@@ -149,17 +152,26 @@ func (p *pvcEvaluator) Handles(a admission.Attributes) bool {
 	if op == admission.Create {
 		return true
 	}
-	if op == admission.Update && utilfeature.DefaultFeatureGate.Enabled(features.ExpandPersistentVolumes) {
-		return true
+	if op == admission.Update && utilfeature.DefaultFeatureGate.Enabled(k8sfeatures.ExpandPersistentVolumes) {
```

Comment:
This "if" block is introduced after rebase. It makes the pvc evaluator handle updates if the "ExpandPersistentVolumes" feature gate is enabled. During the rebase, I made it only handle the update if the pvc is initialized after the update.